qnm_filter/Network.py

```diff
@@ -72,7 +72,8 @@ def __init__(self, **kws) -> None:
 
         self.ra = kws.get('ra', None)
         self.dec = kws.get('dec', None)
-        self.t_init = kws.get('t_init', None)
+        self.peak_time = kws.get('peak_time', None)
+        self.delta_t = kws.get('delta_t', None)
```

Comment:
I've split the t_init attribute into peak_time and delta_t and added t_init as a property. Is useful for the plotter since do not have to calculate delta_t from t_init and peak_time. Should also useful for calculating evidence vs delta_t

I haven't modified the docstrings yet so if you're fine with this change I'll modify them